qcdb/intf_nwchem/options.py

```diff
@@ -600,31 +645,31 @@ def load_nwchem_defaults(options):
     options.add(
         'nwchem',
         RottenOption(
-            keyword='dft_max_ovl',
+            keyword='dft__max__ovl',
```

Comment:
`dft__max_ovl` since `max_ovl` is name of the keyword

Reply:
Thanks for catching that! I thought I got all of them.